session/send.go

```diff
@@ -184,25 +237,21 @@ func (s *Session) composeMessage(recipient, provider string, message []byte, que
 		Payload:   payload,
 		WithSURB:  true,
 	}
-	if query {
-		msg.SURBType = cConstants.SurbTypeKaetzchen
-	} else {
-		msg.SURBType = cConstants.SurbTypeACK
-	}
+	msg.SURBType = cConstants.SurbTypeKaetzchen
```

Comment:
will there be any other kind of SURB? Maybe we can simplify this by removing it completely.